src/exampleTypes/ResultTable.js

```diff
@@ -116,6 +116,18 @@ let Header = withStateLens({ popover: false, adding: false, filtering: false })(
       tree.mutate(filterNode.path, { paused: false })
       F.flip(filtering)()
     }
+    const moveColumn = (computeNewIndex, field, visibleFields) => {
```

Comment:
For performance, it might be worth factoring this method out of the React component so it's not redeclared on each render. The only addition closure dependency it has seems to be `mutate`

Reply:
it's still defined inside of `Header` which means it'll get redeclared every render